.github/workflows/selfhosted.yml

```diff
@@ -0,0 +1,120 @@
+# CI workflow that runs on self hosted cluster
+name: Self-hosted rules
+
+on:
+  workflow_call:
+    inputs:
+      used-labels:
+        description: 'Label names in JSON format'
+        required: true
+        type: string
+      python-version:
+        description: 'Used python version'
+        required: true
+        type: string
+      slurm-job:
+        description: 'Name of the slurm job'
+        required: true
+        type: string
```

Comment:
Can't you hard-code this as well? Or is there a relevant use case for changing it?

Reply:
This is the name of the bash script that we submit to `srun`. I hardcoded it to `selfhosted` and will use this name in all repos.